datadog-agent.yaml

```diff
@@ -122,6 +126,7 @@ pipeline:
     with:
       deps: |-
         golang.org/x/oauth2@v0.27.0
+        golang.org/x/net@v0.36.0
```

Comment:
Upstream already bumped it to 0.37.0 here:
* https://github.com/DataDog/datadog-agent/commit/3eb1856391bab682859c429dc9509529fd639813#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6